controllers/autoscalingrunnerset_controller.go

```diff
@@ -34,29 +56,274 @@ type AutoscalingRunnerSetReconciler struct {
 	Scheme *runtime.Scheme
 }
 
+func getAutoscalerApplicationPodRef(namespace, org, repo, scaleSet, token string) *corev1.Pod {
+	return &corev1.Pod{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Pod",
+			APIVersion: "v1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      fmt.Sprintf("%v-%v", name, scaleSet),
+			Namespace: namespace,
+			Labels:    labels,
+		},
+		Spec: corev1.PodSpec{
+			Containers: []corev1.Container{
+				{
+					Name:  name,
+					Image: image,
+					Env: []corev1.EnvVar{
+						{
+							Name:  "GITHUB_RUNNER_ORG",
+							Value: org,
+						},
+						{
+							Name:  "GITHUB_RUNNER_REPOSITORY",
+							Value: repo,
+						},
+						{
+							Name:  "GITHUB_RUNNER_SCALE_SET_NAME",
+							Value: scaleSet,
+						},
+						{
+							Name:  "GITHUB_TOKEN",
+							Value: token,
+						},
+					},
+				},
+			},
+			RestartPolicy: corev1.RestartPolicyNever,
+		},
+	}
+}
+
 //+kubebuilder:rbac:groups=actions.summerwind.dev,resources=autoscalingrunnersets,verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=actions.summerwind.dev,resources=autoscalingrunnersets/status,verbs=get;update;patch
+//+kubebuilder:rbac:groups=actions.summerwind.dev,resources=autoscalingrunnersets/finalizers,verbs=update
+//+kubebuilder:rbac:groups=*,resources=namespaces;pods,verbs=get;list;watch;create;update;patch;delete
```

Comment:
`kubebuilder:rbac:groups=*,` try `kubebuilder:rbac:groups=,`

Reply:
Fixed to core api group